### DISCUSSION TITLE
Access to Web UI

### DISCUSSION THREAD

**Discussion by titou4307** [haugene/docker-transmission-openvpn#1968] [General]
Hello

I'm a newbee with portainer...
I have made the install of haugene/transmission with NordVPN....

The container Transmission running but it is impossible to access to the web UI

I needeed the help of an expert of the haugene/transmission 

Many thanks for the help

**Comment by pkishino** [comment]
Please see the documentation

**Comment by titou4307** [comment]
Hello pkishino
I'm very newbee, I not expert of this and less of english (I'm french...)

Could you help me more ? I am OK for donation here for the project Haugene or to you ....

Many thanks for your first response and the other

**Reply by pkishino** [reply]
I suspect your LOCAL_NETWORK is not set correctly
check this
https://haugene.github.io/docker-transmission-openvpn/run-container/#3_you_might_not_be_able_to_access_the_web_ui_on_the_first_try

**Comment by titou4307** [comment]
Hello

I'am at my work ..
At the end of the day I will go into the portainer configuration

I think do this : 

Erase all the container Haugene/transmission and Reinstall properly
After this I will add the variable into « Env » and « advanced settings » LOCAL_NETWORK

Many thanks for to take time for me

**Reply by titou4307** [reply]
Hello

Impossible to connect to transmission UI : 

Here the config :

PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
OPENVPN_USERNAME=(erased for the forum)
OPENVPN_PASSWORD=(erased for the forum)
OPENVPN_PROVIDER=NORDVPN
LOCAL_NETWORK=192.168.1.0/24
TRANSMISSION_WEB_UI=combustion
GLOBAL_APPLY_PERMISSIONS=true
TRANSMISSION_HOME=/data/transmission-home
TRANSMISSION_RPC_PORT=9091
TRANSMISSION_DOWNLOAD_DIR=/data/complete
TRANSMISSION_INCOMPLETE_DIR=/data/incomplete
TRANSMISSION_WATCH_DIR=/data/watch
CREATE_TUN_DEVICE=true
ENABLE_UFW=false
UFW_ALLOW_GW_NET=false
UFW_EXTRA_PORTS=
UFW_DISABLE_IPTABLES_REJECT=false
PUID=
PGID=
PEER_DNS=true
PEER_DNS_PIN_ROUTES=true
DROP_DEFAULT_ROUTE=
WEBPROXY_ENABLED=false
WEBPROXY_PORT=8118
WEBPROXY_USERNAME=
WEBPROXY_PASSWORD=
LOG_TO_STDOUT=false
HEALTH_CHECK_HOST=google.com
SELFHEAL=false
REVISION=6ee67f8210d7f9ca17677c2b108390acc91ec5ab

**Comment by pkishino** [comment]
What is the log output? What is your host IP? Did you change the
LOCAL_NETWORK to correctly match your network?

**Comment by titou4307** [comment]
Hello 
In the log I have seen a message with TUN/TAP

My host is 192.168.1.29.....
I have put LOCAL_NETWORK=192.168.1.0/24


I have ticked privilege mode in the portainer container and it possible to access to the web UI :

192.168.1.29:9091/transmission/web

May be it is only the Privileged mode ??

**Reply by pkishino** [reply]
privileged should not be needed if running on a default docker install.. some NAS (like synology) might need this depending on software version.. what OS are you running on your pi? this might be platform/docker version related. perhaps better luck googling docker rpi privileged mode or such. I doubt that this is container related to this one specific.
but at least you know the container works (even if you did have to enable privileged)

**Comment by pkishino** [comment]
Are you running on a NAS?

**Reply by titou4307** [reply]
Hi
No I running the install on a raspberry PI 4 8Go

**Comment by titou4307** [comment]
Hi Pkishino

I use Raspberry Pi OS Lite and connect to this with SSH (or XRDP possible too)

I have made a proper install (since all the test I have modify more and more parameter...)

The install with only the mod of : 

OPENVPN_USERNAME=(erased for the forum)
OPENVPN_PASSWORD=(erased for the forum)
OPENVPN_PROVIDER=NORDVPN
LOCAL_NETWORK=192.168.1.0/24
TRANSMISSION_WEB_UI=(I trying other GUI....)

I have ticked : 
priviliged mode
Active console

I have add too  (id user with root on my Raspberry PI 4) : 
PUID=1001
PGID=1001

And so since the console of the container I could ping Google (8.8.8.8)
I have access to the GUI under 192.168.1.29:9091/transmission/web/

Now I must find : 
How to have/ to link the volume of container Haugene/transmission OpenVPN with volume on the Raspi for to find my download since Transmission ??
And if I have good readind, it needeed to parameter persistence for not lost the download ??

Many thanks for your help

Do you need donation ? 

Titou4307

**Reply by pkishino** [reply]
you can leave the PUID and PGID blank and it will use root as default..
did you not follow the setup in README ? it mentions you need to mount /data, this is used to store the downloads
aka
-v /your/storage/path/:/data

**Comment by titou4307** [comment]
hello
Sorry for to ask a new question ....

I have create a volume into the host (raspberry PI 4) with SSH
this volume is named "docker_data"

I don't know how to link this volume in the host with a volume into a container (into portainer ????)

Can you or could you help me ? I don't understand all the information I find about docker, docker-compose, command line....

Very very many thanks for all the help

**Reply by pkishino** [reply]
You shouldn’t create a volume for your data.
In portainer, edit the container but bind a volume to your file system, see screenshot
![2DEF726C-A3D9-4E6B-9FD3-05693A07A3BB](https://user-images.githubusercontent.com/4121314/135778203-31b5aa3e-496c-4f1f-88e8-3ad710e4a134.jpeg)
In my case I use a docker volume for config but an actual directory on my disk as the bind

**Comment by titou4307** [comment]
Hello Pkishino

I have try your solution and it is OK so many thanks for all the help
I will continue into this Docker solution 

If you need donation for you, your project.... So say me

Best regard

**Reply by Aerya** [reply]
Voici son PayPal : https://www.paypal.com/donate/?cmd=_s-xclick&hosted_button_id=73XHRSK65KQYC
Le principe du don étant de ne pas le demander. Mais puisque tu en parles c'est que tu conviens quand même que son travail et son aide mérite un petit quelque chose de ta part, non ?

**Reply by titou4307** [reply]
Hello @Aerya 

J'ai fait un don à Kristian Haugene (auteur du Docker...) mais je voulais aussi remercier @pkishino (d'ou ma demande de don ....)
I make donation to Augene but i would like to do same to @pkishino 

@Aerya 
Bonjour
pourrais tu m'aider à installer un container Docker non pas depuis le docker Hub mais depuis un lien sur Github.... ?

Merci de ton retour

**Comment by pkishino** [comment]
Hey guys, no worries.
I appreciate the thought. If you can contribute by helping someone else or
making a PR that is more then enough, in this project or some other open
source :)
